docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -27,7 +28,7 @@
 
 Model Optimizer is a cross-platform command-line tool that facilitates the transition between the training and deployment environment, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
 
-Model Optimizer process assumes you have a network model trained using supported deep learning frameworks: Caffe*, TensorFlow*, Kaldi*, MXNet* or converted to the ONNX* format. Model Optimizer produces an Intermediate Representation (IR) of the network, which can be inferred with the [OpenVINO™ Runtime](../OV_Runtime_UG/openvino_intro.md).
+Model Optimizer process assumes you have a network model trained using supported deep learning frameworks: TensorFlow*, PyTorch*, ONNX*, PaddlePaddle*, MXNet*, Caffe*, Kaldi*. Model Optimizer produces an Intermediate Representation (IR) of the network, which can be inferred with the [OpenVINO™ Runtime](../OV_Runtime_UG/openvino_intro.md).
 
 > **NOTE**: Model Optimizer does not infer models. Model Optimizer is an offline tool that runs before the inference takes place.
```

Comment:
maybe "offline tool designed to convert and adjust models, so that they provide optimal performance when inferred with OpenVINO Runtime" ... the note should probably be incorporated in the initial description to avoid repeating content.

Reply:
Done